src/index.ts

```diff
@@ -188,6 +189,10 @@ function fireCommitEvent(target: Element, detail?: Record<string, unknown>): voi
   target.dispatchEvent(new CustomEvent('combobox-commit', {bubbles: true, detail}))
 }
 
+function fireSelectEvent(target: Element): void {
+  target.dispatchEvent(new CustomEvent('combobox-select', {bubbles: true}))
```

Comment:
This can just be an `Event`, doesn't need to be `CustomEvent`.

Reply:
@keithamus indeed. I change that using `Event` now. Thanks! 👍